ci/matlab.yml

```diff
@@ -0,0 +1,56 @@
+# This workflow uses actions that are not certified by GitHub.  They are
```

Comment:
Does this disclaimer appear at the beginning of other templates? Asking this to make sure it has been placed correctly.

Reply:
Yes for example: [gradle.yml](https://github.com/actions/starter-workflows/blob/main/ci/gradle.yml), [rubyonrails.yml](https://github.com/actions/starter-workflows/blob/main/ci/rubyonrails.yml)